paperqa/agents/models.py

```diff
@@ -87,7 +87,7 @@ def set_docs_name(self, docs_name: str) -> None:
 class AnswerResponse(BaseModel):
     model_config = ConfigDict(populate_by_name=True)
 
-    session: PQASession = Field(alias="answer")
+    session: PQASession = Field(alias="session")
```

Comment:
@jamesbraza , this attribute name was changed from `answer` to `session` in  #713 . 
Is it ok to adjust the alias accordingly? 
This solves an error saying that `PQASession` doesn't have a `session` attribute, which I understood was because of the alias.

Reply:
So we need the `alias="answer"` for backwards compatibility, thus we can't make this change

Reply:
If we had `session: ... = Field(alias="session")`, that would make the `alias` redundant to the actual name

Reply:
That's what I thought. But mypy kept giving me this error, saying that there was no `session` attribute. Even vsCode UI was showing the documentation (when you hover over a variable) for `PQSession` with `answer` instead of `session`.

Can it maybe be a cache problem? 
I changed it back to `alias="answer"`, cleaned the pre-commit and will run it again.